var/spack/repos/builtin/packages/py-kymatio/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyKymatio(PythonPackage):
+    """Wavelet scattering transforms in Python with GPU acceleration."""
+
+    homepage = "https://www.kymat.io"
+    url = "https://files.pythonhosted.org/packages/py3/k/kymatio/kymatio-0.3.0-py3-none-any.whl"
```

Comment:
Is there a reason for using the wheel versus building from source using, say `pypi` (https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#id2)?

Reply:
Thanks for the link.